src/Microsoft.TemplateEngine.TemplateLocalizer.Core/LocalizableStrings.resx

```diff
@@ -126,28 +126,36 @@
     <comment>{0} is any string from a json file. Such as "postActions", "myParameter", "author" etc.</comment>
   </data>
   <data name="stringExtractor_log_jsonKeyIsNotUnique" xml:space="preserve">
-    <value>Each child of "{0}" should have a unique id. Currently, the id "{1}" is shared by multiple children.</value>
+    <value>Each child of '{0}' should have a unique id. Currently, the id '{1}' is shared by multiple children.</value>
```

Comment:
is it somehow related to the PR?  or just improvement?

Reply:
Just improvement that I noticed as I was reading the code. Not related to the issue.